src/pyasm/checkin/metadata.py

```diff
@@ -462,13 +462,15 @@ def get_tactic_mapping(my):
 
     def process_tactic_mapping(my, tactic_data, metadata):
 
-        geometry = metadata.get("Geometry")
-        p = re.compile("(\d+)x(\d+)\+(\d+)\+(\d+)")
-        m = p.match(geometry)
-        if m:
-            groups = m.groups()
-            tactic_data['width'] = float(groups[0])
-            tactic_data['height'] = float(groups[1])
+        geometry = str(metadata.get("Geometry"))
```

Comment:
geometry will equal "None" string if it's a NoneType returned  and defeats the if check in the next line